configure.ac

```diff
@@ -1638,7 +1638,7 @@ AS_IF([test x"$with_bfd" != "xno"],
         [BFD_INCLUDE_DIR="/opt/local/include"])
       AS_IF([test -z "$BFD_LIB_DIR"],
         [BFD_LIB_DIR="/opt/local/lib"])],
-    [*-*openbsd*],
+    [*-*-openbsd*|*-*-freebsd*],
```

Comment:
the change of the `openbsd` triple (from `*-*openbsd*` to `*-*-openbsd*`) is intentional, since that's how it is done in the rest of configure.ac.

Reply:
I agree with this assessment - it's clearly a typo from #9134, and any system which managed to report a triple including `fooopenbsd` would clearly not be getting OpenBSD-specific stuff elsewhere in `configure.ac`.